theHarvester/discovery/linkedinsearch.py

```diff
@@ -35,6 +35,7 @@ def get_people(self):
         return rawres.people_linkedin()
 
     def get_links(self):
+        print(self.totalresults)
```

Comment:
Is this needed or was it for testing?

Reply:
just testing. just reverted it.. working on other code now